core/drivers/crypto/caam/include/caam_trace.h

```diff
@@ -34,6 +34,7 @@
 #define DBG_TRACE_RNG	 BIT32(6)  /* RNG trace */
 #define DBG_TRACE_HASH	 BIT32(7)  /* Hash trace */
 #define DBG_TRACE_RSA	 BIT32(8)  /* RSA trace */
+#define DBG_TRACE_CIPHER BIT32(9) /* Cipher dump Buffer */
```

Comment:
tabulation before macro value

Reply:
Macros are automatically aligned by clang-format with tabs and spaces.
I prefer to keep this formatting

Reply:
ok